lib/classes/Swift/Signers/OpenDKIMSigner.php

```diff
@@ -172,7 +168,7 @@ protected function canonicalizeBody($string)
         if (!$this->peclLoaded) {
             return parent::canonicalizeBody($string);
         }
-        if (false && true === $this->dropFirstLF) {
+        if (true === $this->dropFirstLF) {
```

Comment:
We need this. The CRLF being dropped is added in [SimpleMimeEntity::bodyToByteStream()](https://github.com/swiftmailer/swiftmailer/blob/master/lib/classes/Swift/Mime/SimpleMimeEntity.php#L540).